openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -56,27 +56,17 @@ def _do_third_party_auth(request):
             u"with backend_name {backend_name}".format(
                 username=username, backend_name=backend_name)
         )
-        message = _(
+        message = Text(_(
             u"You've successfully logged into your {provider_name} account, "
-            u"but this account isn't linked with an {platform_name} account yet."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += _(
+            u"but this account isn't linked with your {platform_name} account yet. {blank_lines}"
             u"Use your {platform_name} username and password to log into {platform_name} below, "
```

Comment:
"to sign in to {platform_name}"

Reply:
Done in https://github.com/edx/edx-platform/pull/20253.